test/unit-tests/test-output/_utils.ts

```diff
@@ -0,0 +1,5 @@
+export function getDummyStream(): any {
```

Comment:
I wasn't sure @jamesrichford, it creates a pretty shallow object but I suppose it could be considered a builder. What do you think?